### DISCUSSION TITLE
p/hugo-obsidian-stagic-blog1/

### DISCUSSION THREAD

**Discussion by giscus** [eatmoreduck/eatmoreduck.github.io#3] [Announcements]
# p/hugo-obsidian-stagic-blog1/

笔记软件选择原则 拒绝云服务，数据留存本地，本机运行，使用github/gitee 私仓来进行笔记储存 拒绝专有格式，使用通用的标记型语言 开源 跨平

https://blog.xiaohuangyu.space/p/hugo-obsidian-stagic-blog1/

**Comment by eatmoreduck** [comment]
test giscus comment! lol
